go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/opencontainers/runc
 
-go 1.22
+go 1.22.0
```

Comment:
?

Reply:
Also a side-effect from go get, but I can manually override it.

Reply:
It turns out, as mentioned on my other comment:
```
go: golang.org/x/exp@v0.0.0-20240909161429-701f63a606c0 requires go@1.22.0, but 1.22 is requested
```

so I reckon this line will need to stay unchanged, otherwise the deps check [fails](https://github.com/opencontainers/runc/actions/runs/10799570842/job/29955598612?pr=4397).

If that's still an issue, let me know so we can think of something else.